install_config/master_node_configuration.adoc

```diff
@@ -1013,3 +1013,213 @@ To launch a node server using a node configuration file:
+
+[NOTE]
+====	
+Messages with FATAL, ERROR, WARNING and some INFO severities appear in the logs regardless of the log configuration.
+====
+
+You can view logs for the master or the node system using the following command:
+
+----
+# journalctl -r -u <journal-name>
+----
+	
+Use the -r option to show the newest entries first.
+
+For example:
+
+----
+# journalctl -r -u atomic-openshift-master.service
+# journalctl -r -u atomic-openshift-node.service
+----
+
+To change the logging level:
+
+. Edit the `/etc/sysconfig/atomic-openshift-master` file for the master or `/etc/sysconfig/atomic-openshift-node` file for the nodes.
+. Enter a value from the *Log Level Options* table above in the `OPTIONS=--loglevel=` field.
++
+For example:
++
+----
+OPTIONS=--loglevel=4
+----
+. Restart the master or node host as appropriate:
+
+----
+# systemctl restart atomic-openshift-master
+# systemctl restart atomic-openshift-node
+----
+
+After the restart, all new log messages will conform to the new setting. Older messages do not change.
+
+[NOTE]
+====
+The default log level can be set using the Advanced Install. For more information, see xref:../install_config/install/advanced_install.adoc#cluster-variables-table[Cluster Variables].
+====
+
+////
+
+.Log Level Options
+[cols="3a,6a",options="header"]
+|===
+
+|Option |Description
+
+|`0`
+|Errors (E), Warnings (W), and some Information (I) messages are logged.
+
+|`2`
+|Errors (E), Warnings (W), and the following informational messages are logged:
+
+ * programmer errors
+ * logging extra info about a panic
+ * CLI argument handling
+ * information about config (listening on X, watching Y)
+ * errors that repeat frequently that relate to conditions that can be corrected (pod detected as unhealthy)
+ * logging HTTP requests and their exit code
+ * system state change events (killing pod)
+ * controller state change events (starting pods)
+ * scheduler log messages
+
+This level is equal to glogV(1), glogV(2), and glogV(3) and is the default level.
+
+|`4`
+|Errors (E), Warnings (W), and the and additional informational messages for debugging purposes.
+
+ * Kubernetes fit failures
+
+ This level is equal to glogV(4).
+
+|`6`
+|API GET requests and URLs are logged.
+
+Level 0 (E), (W), and (I) messages are also logged.
+
+|`7`
+|The followng API details are logged for each GET request:
+
+ * GET request URL
+ * Request header
+ * Response status
+
+Level 0 (E), (W), and (I) messages are also logged.
+
+|`8`
+|The followng API details are logged for each GET request:
+
+ * GET request URL
+ * Request header
+ * Response status
+ * Response header
+ * Response body
+
+Level 0 (E), (W), and (I) messages are also logged.
+
+|===
+
+////
+
+The following examples are excerpts from a master journald log at various log levels. Timestamps and system information has been removed from these examples.
+
+.Excerpt of journalctl -u atomic-openshift-master.service output at loglevel=0
+====
```

Comment:
You can take out all the `=====` blocks around these examples, we aren't using them anymore since they don't show up differently in the Portal build.